src/asound/bluealsa-pcm.c

```diff
@@ -345,10 +348,19 @@ static void *io_thread(snd_pcm_ioplug_t *io) {
 
 fail:
 
+	if (pcm->ba_pcm_fd != -1) {
```

Comment:
Is it required to close file descriptors here? With the `connected` flag, we can relay on the `bluealsa_hw_free()` being called by the ALSA library - right now there is no function which checks for `ba_pcm_fd == -1`.